src/components/Settings/Settings.scss

```
@@ -1957,6 +1957,108 @@
+    padding: 0 2px;
+    margin: 0;
+    color: inherit;
+    font: inherit;
+    cursor: pointer;
+
+    // A base-select control carries a UA minimum height of 24px — a touch
+    // target — against the 13.85px of text in the links beside it, and its
+    // chevron holds a full line box on top of that. The row's boxes were
+    // centred on each other either way, but the eye aligns a line of links on
+    // their TEXT, and a taller box centres its text somewhere else.
+    //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)